tools/hangar/src/meta_comment.ts

```diff
@@ -25,7 +25,14 @@ export interface MetaComment {
    *
    * @default - Will not be skipped
    */
-  skipPlatforms?: typeof process.platform[];
+  skipPlatforms?: (typeof process.platform)[];
+
+  /**
+   * Should the test be skipped at all platform, all environments
+   *
+   * @default - Will not be skipped
+   */
+  skipTest?: boolean;
```

Comment:
```suggestion
  skip?: boolean;
```

Also is this used in the PR? If not we can probably omit it, I'm not sure we want to include tests that are always skipped.

Reply:
Cool, I'll change it in my next PR (which includes SDK spec tests to skip- for example a test for queue.pop- that isn't an existent method yet)